src/components/EPubsCard/EPubsCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styles from './EPubsCard.module.scss'
+import LinkTo from 'components/util/LinkTo/LinkTo'
+
+type PropTypes = {
+  query: string
+}
+
+const EPubsCard = ({ query }: PropTypes) => {
+  const ePubsSearch = `https://search.usa.gov/search?affiliate=afpw_epubs&query=${query}`
```

Comment:
I saw Noah's comments about using either url to perform the epubs search -- @jbecker01 @MKDale do either of you have a preference? Asking because the results ui is a bit different depending on which one we choose. Apologies if this was already decided!

Reply:
I believe we want to update this to use the URL that points to the new ePubs interface. @MKDale does that sound right to you?